### ISSUE TITLE
Black screen, got no images

### ISSUE THREAD

**Issue by angrypiggy** [LI01/linux_camera_tool#21]
I have configured the environment following your tutorial and after typing ./leopard -d 2 the control panel was poped out but I still got no images.

**Comment by andrejlevkovitch** [comment]
@angrypiggy Hi! Did you find solution of the problem? I have same

**Comment by angrypiggy** [comment]
> @angrypiggy Hi! Did you find solution of the problem? I have same

No, I did not end up with a solution. I contacted the staff and they say it should be a voltage problem. Sometimes I got the camera to work, but most of the time I only got black screen. I suspect it is due to bad connection, but I had no clue how to fix and stabilize it.
